LICENSE

```diff
@@ -1,5 +1,5 @@
-Copyright (c) 2006-2018, Knut Reinert & Freie Universitaet Berlin
-Copyright (c) 2016-2018, Knut Reinert & MPI Molekulare Genetik
+Copyright (c) 2006-2019, Knut Reinert & Freie Universitaet Berlin
+Copyright (c) 2016-2019, Knut Reinert & MPI Molekulare Genetik
 All rights reserved.
```

Comment:
Can you check the existing "new" copyright headers? We changed them to include Umlauts and also write "MPI für ..."

Reply:
Thanks for catching this. Was only the case in this file.